notebooks/hive/PyHive.ipynb

```diff
@@ -0,0 +1,726 @@
+       "        <td>Mon May 19 00:00:00 EDT 2008</td>\n",
+       "        <td>266000.0</td>\n",
+       "        <td>38.671993</td>\n",
+       "        <td>-121.32434</td>\n",
+       "        <td>95608</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "        <td>4622 MEYER WAY</td>\n",
+       "        <td>CARMICHAEL</td>\n",
+       "        <td>CA</td>\n",
+       "        <td>4</td>\n",
+       "        <td>2</td>\n",
+       "        <td>1559.0</td>\n",
+       "        <td>Residential</td>\n",
+       "        <td>Mon May 19 00:00:00 EDT 2008</td>\n",
+       "        <td>285000.0</td>\n",
+       "        <td>38.64913</td>\n",
+       "        <td>-121.31067</td>\n",
+       "        <td>95608</td>\n",
+       "    </tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "[('5332 SANDSTONE ST', 'CARMICHAEL', 'CA', 3, 1, 1152.0, 'Residential', 'Wed May 21 00:00:00 EDT 2008', 181872.0, 38.662106, -121.31394, 95608),\n",
+       " ('5907 ELLERSLEE DR', 'CARMICHAEL', 'CA', 3, 1, 936.0, 'Residential', 'Wed May 21 00:00:00 EDT 2008', 200000.0, 38.664467, -121.32683, 95608),\n",
+       " ('4010 ALEX LN', 'CARMICHAEL', 'CA', 2, 2, 1326.0, 'Condo', 'Wed May 21 00:00:00 EDT 2008', 250134.0, 38.637028, -121.312965, 95608),\n",
+       " ('5925 MALEVILLE AVE', 'CARMICHAEL', 'CA', 4, 2, 1120.0, 'Residential', 'Tue May 20 00:00:00 EDT 2008', 189000.0, 38.666565, -121.325714, 95608),\n",
+       " ('2109 HAMLET PL', 'CARMICHAEL', 'CA', 2, 2, 1598.0, 'Residential', 'Tue May 20 00:00:00 EDT 2008', 484000.0, 38.602753, -121.32932, 95608),\n",
+       " ('5709 RIVER OAK WAY', 'CARMICHAEL', 'CA', 4, 2, 2222.0, 'Residential', 'Tue May 20 00:00:00 EDT 2008', 582000.0, 38.602463, -121.33098, 95608),\n",
+       " ('7032 FAIR OAKS BLVD', 'CARMICHAEL', 'CA', 3, 2, 1245.0, 'Condo', 'Mon May 19 00:00:00 EDT 2008', 139500.0, 38.628563, -121.3283, 95608),\n",
+       " ('7110 STELLA LN Unit 15', 'CARMICHAEL', 'CA', 2, 2, 1000.0, 'Condo', 'Mon May 19 00:00:00 EDT 2008', 182000.0, 38.637398, -121.30006, 95608),\n",
+       " ('5847 DEL CAMPO LN', 'CARMICHAEL', 'CA', 3, 1, 1713.0, 'Residential', 'Mon May 19 00:00:00 EDT 2008', 266000.0, 38.671993, -121.32434, 95608),\n",
+       " ('4622 MEYER WAY', 'CARMICHAEL', 'CA', 4, 2, 1559.0, 'Residential', 'Mon May 19 00:00:00 EDT 2008', 285000.0, 38.64913, -121.31067, 95608)]"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "select * \n",
+    "FROM sacramento_properties \n",
+    "WHERE zip=95608\n",
+    "LIMIT 10"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result of the query can also be stored in a Python dictionary. This allows us to use the vast library of python library to visualise the data and build reports.\n",
```

Comment:
@SirOibaf `The query result can also be stored in a Python dictionary. This enables us to choose from a plethora of Python libraries available online, to visualize data and build BI reports`.